oemr-selenium/test_utils.py

```diff
@@ -14,3 +16,29 @@ def get_user():
 
 def get_pass():
```

Comment:
please remove existing functions, if you are replacing it with other functions in utls.

Reply:
Removed existing functions.